docs/configuration/integrations/gitlab-enterprise.md

```diff
@@ -9,10 +9,10 @@ Follow the instructions below to set up the Codacy integration with GitLab Cloud
     You'll need to add the following "Redirect URI", making sure to update both your protocol to use either HTTP or HTTPS and your hostname as well. Keep in mind that this field is case sensitive.
 
     ```
-    https://codacy.example.com/login/GitLab
-    https://codacy.example.com/add/addPermissions/GitLab
-    https://codacy.example.com/add/addProvider/GitLab
-    https://codacy.example.com/add/addService/GitLab
+    https://codacy.example.com/login/GitLabEnterprise
```

Comment:
I found this was wrong

Reply:
It was fixed also here https://github.com/codacy/chart/pull/376